src/pages/EditMonitor.vue

```diff
@@ -61,6 +61,9 @@
                                         <option value="mqtt">
                                             MQTT
                                         </option>
+                                        <option value="nut">
+                                            NUT
```

Comment:
NUT is not as searchable as the other items.
What do you think of the following (name is from [their website](https://networkupstools.org/projects.html)):

```suggestion
                                            Network UPS Tools
```